### ISSUE TITLE
Web push activity notifications have low contrast when using dark mode on desktop

### ISSUE THREAD

**Issue by salmart-dev** [nextcloud/server#58695]
## Summary

The notifications a user receives when subscribing for updates to e.g. share modifications have an accessibility problem: when the user uses web push notifications and their OS is in dark mode, the activity icon in the notification is black, which combined with the dark background of the notification makes it really hard to see.

We should discuss to see which solution we can implement for this.

## How to reproduce

1. Use a OS with Dark Theme support or with a theme which makes the background of notifications dark
2. Enable the `notifications` and `activity` apps on a Nextcloud instance
3. Enable HTTPS in your instance (in Julius' dev setup [can be done easily](https://juliusknorr.github.io/nextcloud-docker-dev/basics/ssl/#https))
4. Open a browser tab pointing at your instance using `https://...`
5. Make sure to accept the browser's request asking for permissions to display notifications
6. With User A, navigate to "Personal Settings -> Notifications" and enable push notifications for the action you wish to test
7. With User B, trigger the action that will produce a notification for User B
8. Observe the received notification (from your browser!) it will have a black icon for the activity, which is barely visible.

#### KDE + Firefox

<img width="402" height="152" alt="Image" src="https://github.com/user-attachments/assets/3b538d55-f330-4b5d-ab9b-00f756d0ec18" />

#### Windows 11 + Firefox

<img width="348" height="254" alt="Image" src="https://github.com/user-attachments/assets/29f9c5cb-5932-48bc-9552-ae9f682b67fa" />

**Comment by salmart-dev** [comment]
@nextcloud/designers any comment on this?
